src/styles/reset.css

```diff
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
do
```suggestion
```
do not reset margin or padding for all selectors. reset only tags, which necessary (ul, p, body, h1...)